src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java

```diff
@@ -29,9 +30,14 @@ private ScannedDocumentsHelper() {
 
     @SuppressWarnings("unchecked")
     public static List<Document> getDocuments(CaseDetails caseDetails) {
-        List<Map<String, Object>> data = (List<Map<String, Object>>) caseDetails.getData().get("scannedDocuments");
+        Map<String, Object> caseData = caseDetails.getData();
+        Assert.notNull(caseData, "Case Data must not be null");
```

Comment:
where is `getDocuments` used?
Do we actually want to throw `IllegalArgumentException` or return an empty list??

Reply:
I guess it's used for extracting existing documents from case data before adding more. As for the error, maybe we should just treat it as an empty case to add fields to? I don't know myself, just asking.

Reply:
It's being used when attaching evidence to an existing case - `AttachDocsToSupplementaryEvidence.buildCaseDataContent()`. 
we want to throw `IllegalArgumentException` as ccd data is saved as empty collection in database when no scanned documents or case data is not provided

Reply:
After discussing with probate, some user roles (eg: solicitor) don't have access to scannedDocuments field. In such cases, scannedDocuments are null in the ccd case. So I'm changing the code to return empty list